### ISSUE TITLE
[Suggestion/Feature Request] Knives cut Cactus quickly

### ISSUE THREAD

**Issue by Bentcheesee** [vectorwing/FarmersDelight#1029]
Since V1.2.5, Melons and Pumpkins can now be broken quickly by knives, so would it be too far of a stretch for Cactus to be as well?

It's just a small little thing I thought of about seven minutes ago and I didn't see anything else that asked about it so I wanted to throw the idea out there for you to grab if you do so choose.

**Comment by vectorwing** [comment]
Yeah, that sounds about right. I'll toss it in for the 1.2.7 release. 👍
